command_stream.py

```diff
@@ -1,21 +1,20 @@
 from serverless_sdk import tag_event
-from clients import lambdas
-from utils.kinesis import get_payloads_from_kinesis_payload
+from clients import sqs
+from utils.kinesis import get_payloads_from_kinesis_event
 
 
 COMMAND_TYPES = {"INBOUND_SMS"}
 
 
 def handle_command(raw_event, context):
     tag_event("command_stream", "handle_command", raw_event)
-    events = get_payloads_from_kinesis_payload(raw_event)
+    events = get_payloads_from_kinesis_event(raw_event)
     tag_event("command_stream", "commands", events)
 
-    for event in events:
-        response = get_response(event)
+    responses = [get_response(event) for event in events]
 
-        # eventually: write to dynamodb dialog log
-        lambdas.invoke_send_message(response)
+    # temporary for now. eventually this should write to the dialog log
```

Comment:
Confirming that we are in sync on how this will eventually work.

Inbound SMS -> command queue -> dialog agent -> user event stream -> SQS -> twilio 

Just double checking. If we see it differently then let’s talk in the morning and iron it out.

Reply:
Correct - that's my understanding. Is the user event stream -> sqs going to be done via a dynamo stream?

Reply:
exactly. dialog agent -> dynamodb -> dynamodb stream